ext/opcache/jit/zend_jit_arm64.dasc

```diff
@@ -11869,6 +11883,21 @@ static int zend_jit_fetch_obj(dasm_State          **Dst,
 		} else {
 			|	IF_UNDEF REG2w, >5
 		}
+		if (opline->opcode == ZEND_FETCH_OBJ_W && (prop_info->flags & ZEND_ACC_READONLY)) {
+			|	IF_NOT_TYPE REG2w, IS_OBJECT_EX, >4
```

Comment:
Why is this IS_OBJECT_EX instead of IS_OBJECT?

In the x86 jit code, I see IS_OBJECT being used, and similarly, I see the single-bit flags such as IS_RESOURCE used elsewhere for arm with the IF_NOT_TYPE macro?

```
			|	IF_NOT_TYPE TMP1w, IS_REFERENCE, >1
```